Revise NNS Intro (#2441)
dfinity/portal: docs/tokenomics/nns/nns-intro.md

# Network Nervous System (NNS)

## Overview​

> The [NNS](https://nns.ic0.app/) is the decentralized autonomous organization (DAO) that governs the Internet Computer Protocol. 

Reviewer: My preference is to not call the Frontend "the NNS" because others from the community might also build a frontend for the NNS. So I would add a sentence, "the on-chain frontend that allows you to interact with it is here" and link to the frontend there

Author: That's a good point. I will also note to provide more documentation on how to interact with the governance canister in the future :)

It is a fully on-chain, decentralized organization and is, for instance, responsible for making protocol-level upgrades to continuously improve the Internet Computer Protocol. 

> It operates under a liquid democracy in which those who have staked their ICP or [ICP neuron holders vote](../../concepts/governance.md) vote on proposals that shape the development of the Internet Computer. 

Reviewer: I think
> those who have staked their ICP or [ICP neuron holders vote]

is confusing as it is not clear whether the "or" refers to two different concepts..
How about "those who have staked their ICP in a _neuron_ 

(and I think "vote" might be duplicated?)

Author: Good catch! Yes, I think that your suggestion is more clear!

Examples of proposals include:
1. upgrade the protocol and software used by the node machines that host the network
2. create new subnet blockchains to increase network capacity

> 3. split subnets to divide their load

Reviewer: I think it is fine but we rarely (never) did this, so maybe not a super representative example

> 4. configure economic parameters that control how much must be  paid by users for compute capacity

Reviewer: ```suggestion
4. configure economic parameters that determine the cost for compute capacity
```

> 5. In extreme situations, it can freeze malicious canister smart contract software to protect the network; and many other things. 

Reviewer: do we really want to point out this controversial topic? I would rather leave that out.

If you want one more once could mentioned the proposals that update the DAO itself (with voting rules etc)

A proposal passes based on either:
- **Absolute Majority** - At any point, even before the voting period ends, if an absolute majority (more than half of the total voting power) has voted "yes," then the proposal is adopted. If an absolute majority has voted "no," then the proposal is rejected.

- **Simple Majority** - When the voting period ends, if a simple majority (more than half of the cast votes) has voted "yes" and the number of these yes votes constitutes at least 3% of the total voting power, then the proposal is adopted. Otherwise, the proposal is rejected.

> The governance voting algorithm also applies to the "wait for quiet" period, which decides on proposals quickly if all voters agree, but increases the time that neurons can vote for more controversial proposals. Depending on the votes, the voting period can be dramatically increased.

Reviewer: I think this is a bit confusing. How about: 

```suggestion
For determining how long proposals are open for voting, the governance uses a so called "Wait-for-quiet" algorithm. This ensures that proposals are decided quickly if all voters agree, but have a longer voting period if they are controversial. Most proposals start with 4 days of voting period that can be extended up to 8 days.
```

> Once such a proposal is accepted, it is autonomously executed.

Reviewer: ```suggestion
Once a proposal is adopted, it is autonomously executed on-chain.
```

(note that this is not true for motion proposals so it would be more correct to say "for most" or "with the exception of..")

> While other blockchains take weeks or months to upgrade (sometimes called hard fork) and typically require substantial manual work and coordination to do so, ICP upgrades itself every week (https://dashboard.internetcomputer.org/releases). Its ability to upgrade and iterate is a comparative "superpower."

Reviewer: ```suggestion
While other blockchains take weeks or months to upgrade and typically require substantial manual work and coordination to do so, ICP upgrades itself every week (https://dashboard.internetcomputer.org/releases). Its ability to upgrade and iterate is a comparative "superpower."
```

I would remove this: it is only a hard fork if a part of the network upgrades but the other part does not. So not all upgrades can be called that way I would say.

## Proposals 
Participants submit proposals to the NNS. Refer to the proposal submission requirements and types at [Proposal Requirements](./proposal-requirements.md).

> Proposals are adopted or rejected based on voting activity by “neurons” that network participants have created.

Reviewer: I wonder if neurons need to be introduced in 1 sentence before mentioning them here.
they appeared above but were not really "Defined" I think

Author: I thnk a better flow is:

- Participation Rewards
- Proposals
- Participation

Thank you for the suggestion!

## Participation Rewards
Network governance on the Internet Computer is incentivized through a system called "neurons".

> ICP token holders can create a “neuron” by locking balances of ICP up to eight years that are hosted on a ledger inside the NNS. 

Reviewer: ```suggestion
ICP token holders can create a “neuron” by staking ICP up to eight years that are hosted on a ledger inside the NNS. 
```

> When a user creates a neuron, the locked balance of ICP can only be unlocked by disbursing (“destroying”) the neuron. Each neuron is configured with “dissolve delay.” or setting that determines the minimum amount of time that you have to wait before you can unlock your original locked ICP utility tokens. Once a neuron has been placed into “dissolve mode,” its dissolve delay falls over time, rather like a kitchen timer, until it reaches zero, whereupon its owner can perform a final disburse action to unlock the balance of ICP. 

Reviewer: Why is this explanation of neurons under the section "proposals"? (Not saying this can't work, but I didn't quite understand this structure)

```suggestion
When a user creates a neuron, the staked ICP can only be taken out of the neuron again by disbursing the neuron. Each neuron is configured with a “dissolve delay” that determines the minimum amount of time that you have to wait before you can unlock your staked ICP utility tokens. Once a neuron has been placed into “dissolving mode,” its dissolve delay falls over time, rather like a kitchen timer, until it reaches zero, whereupon its owner can perform a final disburse action to unlock the balance of ICP. 
```

Author: Thank you, I moved it under Participation Rewards!

> Neuron owners are economically incentivized to vote in the best interests of the network by locking their ICP tokens for longer periods.  

Reviewer: ```suggestion
Neurons with a longer dissolve delay have more voting power and get more rewards so that they are economically incentivized to vote in the best long-term interests of the network.  
```

> The number of rewards disbursed to a neuron derives from such factors as the size of the locked balance, the minimum lockup period remaining (the “dissolve delay”), the neuron’s “age,” the proportion of possible votes it has participated in, and the sum of voting activity across all neurons, since the overall total rewards disbursed is capped and must be divided between voters.

Reviewer: ```suggestion
The number of rewards that neuron receives derive from factors as the amount of staked tokens, the lockup period remaining (the “dissolve delay”), the neuron’s “age”, the proportion of possible votes it has participated in, and the sum of voting activity across all neurons, since the overall total rewards are capped and must be divided between voters.
```

> It is expected that a large proportion of the overall supply of ICP will be locked to earn rewards to secure the Internet Computer’s network governance from attacker to acquire a sufficiently large stake to gain significant influence. 

Reviewer: I am not sure if I would make this claim.
First, I think we shouldn't imply that by just staking you get rewards (you actually have to vote to get rewards). Second, I am not sure we want to say this is an expectation.
We could maybe just say that at the moment it is a reality (if this is true)? Or maybe omit this?

For more information on neurons, refer to the [What are Neurons section in the NNS quickstart](https://internetcomputer.org/docs/current/tokenomics/token-holders/nns-app-quickstart).
For more information on staking rewards, refer to [Staking and Voting Rewards](https://internetcomputer.org/docs/current/tokenomics/nns/nns-staking-voting-rewards).

## Participation

> Neuron owners face challenges in manually voting on every NNS proposal due to high submission volumes and a potential lack of expertise. The NNS allows automatic voting through delegation where neurons automatically vote by following others for all proposals or a specific proposal topic. This ensures efficient decision-making without requiring constant manual input.

Reviewer: ```suggestion
Neurons can manually vote on all proposals. 
If they cannot manually vote on all proposals due to high submission volumes or a lack of expertise, neurons can delegate their vote, called _following_, for specific proposal topic. This ensures efficient decision-making without requiring constant manual input.
```

> It is expected that a large proportion of the overall supply of ICP will be locked to earn rewards. This secures the Internet Computer’s network governance, by making it both difficult and exorbitantly expensive for an attacker to acquire a sufficiently large stake to gain significant influence. Since neuron owners may wish to maximize their rewards by voting on all proposals, most neurons will either be actively managed or configured to follow other neurons so that they can vote automatically.

Reviewer: I think this is a repetition of above. As already mentioned there I am not sure if we want to mention this.
Do you think this is important? What is the main message we want to deliver here?

Author: I think we were aiming to convey this idea:

"Since the incentive to stake ICP in a neuron is high, it is difficult and exorbitantly expensive for an attacker to acquire a sufficiently large stake to gain significant influence."

Does this sound better?
